src/query.ts

```diff
@@ -75,6 +90,16 @@ export class Query {
     return this._getFilter(this._excludedFilters);
   }
 
+  public get sort(): GenericObject {
+    const result: InputExtendedSortValue[] = [];
+    this._sort.forEach((value, key) => {
+      const sortValue: GenericObject = {};
```

Comment:
Seems that this should be `InputExtendedSortValue` instead of `GenericObject`.

Reply:
done